source/_patterns/components/view-selector/view-selector.hbs

```diff
@@ -0,0 +1,11 @@
+<div class="pf-view-selector" role="group" aria-label="View selector">
+  <button class="btn btn-link pf-view-selector__btn-link" aria-label="Table view">
```

Comment:
@srambach had included this question in her original PR post:
- should the buttons actually be a basic pattern?

I think it's very likely that we would use something like this in more places.

Is it possible to create a button pattern that could display an icon instead of text, and be reused here and also in the dropdown pattern?